src/donationForm.js

```diff
@@ -413,19 +406,6 @@ const DonationForm = (props) => {
                             </div>
                             <form onSubmit={handlePaymentSubmit}>
                                 <div className={`donation-form-payment-intent ${css(styles.stripePaymentWrap)}`}></div>
-                                {error && (
```

Comment:
@JasonTheAdams this error output needs to be in place. Without it, messages from the Stripe API about the payment method in use are not displayed. For instance:

**Current develop branch**
<img width="730" alt="2022-03-03_11-48-32" src="https://user-images.githubusercontent.com/1571635/156642122-1a31f57a-1e35-4cf5-bae0-f5b09c3ab5a4.png">

**In this Branch**
<img width="711" alt="2022-03-03_11-51-19" src="https://user-images.githubusercontent.com/1571635/156642130-3414dc13-335f-47b0-9aa5-822a394b6238.png">